src/docs/sphinx/index.rst

```diff
@@ -33,6 +33,21 @@ Serac is hosted on GitHub `here <https://github.com/LLNL/serac>`_. Serac uses gi
    # for some commands
    $ git clone --recursive https://github.com/LLNL/serac.git
 
+Preparing Windows WSL/Ubuntu 18.04 for Serac installation
+---------------------------------------------------------
+
+For faster installation of the Serac dependencies via Spack on Windows WSL/Ubuntu 18.04 systems, install cmake, MPICH, openblas, and OpenGL 
+using the following commands:
+
+.. code-block:: bash
+   $ sudo apt-get update
+   $ sudo apt-get upgrade
+   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev
+   $ sudo ln -s /usr/lib/x86_64-linux-gnu/* /usr/lib
```

Comment:
I wonder if this should be logged as a bug to spack...

Reply:
sigh... probably.